perfmetrics/scripts/ml_tests/pytorch/run_model.sh

```diff
@@ -46,13 +53,14 @@ EOF
 echo "Created config-file at "$config_filename
```

Comment:
Master is different in adding metadata-cache-ttl sec 
https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/ml_tests/pytorch/run_model.sh

My changes are okay of changing bucket name and removing epochs.
https://github.com/GoogleCloudPlatform/gcsfuse/pull/1610

Reply:
Yes this is where I was unsure. @gargnitingoogle, can you confirm if type-cache-ttl, stat-cache-ttl flags should be removed? Can you also check if any other changes are required?

Reply:
> can you confirm if type-cache-ttl, stat-cache-ttl flags should be removed?

Yes, we should not use type-cache-ttl or stat-type-ttl in the read_cache_release branch now as they are to be deprecated and replaced with ttl-secs config. So, the change on line 56-57 (on RHS) should be reverted.

> Can you also check if any other changes are required?

All other changes of mine look fine to me.